xds/third_party/envoy/import.sh

```diff
@@ -75,19 +75,19 @@ do
     # Import mangling.
     -e 's#import "gogoproto/gogo.proto";##'
     # Remove references to gogo.proto extensions.
-    -e 's#option (gogoproto\.[a-z_]\+) = \(true\|false\);##'
-    -e 's#\(, \)\?(gogoproto\.[a-z_]\+) = \(true\|false\),\?##'
+    -e 's#option \(gogoproto\.[a-z_]+\) = (true|false);##'
+    -e 's#(, )?\(gogoproto\.[a-z_]+\) = (true|false),?##'
     # gogoproto removal can result in empty brackets.
     -e 's# \[\]##'
     # gogoproto removal can result in four spaces on a line by itself.
     -e '/^    $/d'
   )
-  sed -i "${commands[@]}" "$f"
+  sed -E -i '' "${commands[@]}" "$f"
```

Comment:
This `-i` won't work for GNU sed, since it expects the argument to be next to the `-i`. Apparently `-i''` works for OS X, and that would also work for Linux, although I have no clue how it works for OSX.